admin_manual/configuration_database/linux_database_configuration.rst

```diff
@@ -49,46 +49,11 @@ If you decide to use a MySQL or MariaDB database, ensure the following:
 
   [mysqld]
   ...
-  transaction_isolation = READ-COMMITTED
-  binlog_format = ROW
-  ...
-
-Your :file:`/etc/mysql/my.cnf` could look like this:
-
-::
-
-  [server]
-  skip_name_resolve = 1
-  innodb_buffer_pool_size = 128M
-  innodb_buffer_pool_instances = 1
-  innodb_flush_log_at_trx_commit = 2
-  innodb_log_buffer_size = 32M
-  innodb_max_dirty_pages_pct = 90
-  query_cache_type = 1
-  query_cache_limit = 2M
-  query_cache_min_res_unit = 2k
-  query_cache_size = 64M
-  tmp_table_size= 64M
-  max_heap_table_size= 64M
-  slow_query_log = 1
-  slow_query_log_file = /var/log/mysql/slow.log
-  long_query_time = 1
-
-  [client-server]
-  !includedir /etc/mysql/conf.d/
-  !includedir /etc/mysql/mariadb.conf.d/
-
-  [client]
-  default-character-set = utf8mb4
-
-  [mysqld]
   character_set_server = utf8mb4
   collation_server = utf8mb4_general_ci
   transaction_isolation = READ-COMMITTED
   binlog_format = ROW
-  innodb_large_prefix=on
-  innodb_file_format=barracuda
-  innodb_file_per_table=1
```

Comment:
innodb_file_per_table=1 is important afaik. We need it for utf8mb4 as per https://github.com/nextcloud/documentation/blob/82ad366697c2ba982498bc71fea9d3a8e4215d39/admin_manual/configuration_database/mysql_4byte_support.rst. 

Could you add it back and set a link to the mysql 4byte page?

Reply:
I think the docs are dated. AFAIK `innodb_file_per_table=1` is the default in all the db versions we support:

https://dev.mysql.com/doc/refman/8.0/en/innodb-parameters.html#sysvar_innodb_file_per_table

Looks like on has been default since MySQL 5.6 and at least MariaDB Server 10.0.

It's also deprecated, at least in MariaDB, but that's a recent change and not relevant as long as we're supporting 10.x:

https://mariadb.com/kb/en/innodb-system-variables/#innodb_file_per_table

Reply:
:+1: